src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.ToJson.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
+    {
+        private static bool ToJson(
```

Comment:
What does returning true/false here mean?

Reply:
Currently not used until we add async. True=continue. False=done. The callee then if True was returned will write to the steam and async flush